src/components/EmptyMessage.js

```diff
@@ -1,13 +1,19 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import { RegularText } from '.';
+import { RegularText, Wrapper, WrapperWithOrientation, SafeAreaViewFlex } from '.';
 
 export const EmptyMessage = ({ title }) => {
   return (
-    <RegularText small center>
-      {title}
-    </RegularText>
+    <SafeAreaViewFlex>
+      <WrapperWithOrientation>
+        <Wrapper>
```

Comment:
why is this needed now and moved here? does it still fit for our cases?

Reply:
all right, now i understand the reason. then i also think it is better be placed here, thanks.

Reply:
If wrapping is deleted | If wrapping is not deleted |
|--|--
![Simulator Screen Shot - iPhone 13 - 2022-01-27 at 13 27 05](https://user-images.githubusercontent.com/11755668/151340776-0eee5aeb-0d98-465d-9a8d-6d95d6a08ada.png) | ![Simulator Screen Shot - iPhone 13 - 2022-01-27 at 13 08 18](https://user-images.githubusercontent.com/11755668/151340999-30d73133-4284-4d6f-b161-5872be0e7e01.png) |
IndexScreen.js | IndexScreen.js |